src/SoftDeleteable/Mapping/Validator.php

```diff
@@ -29,6 +29,7 @@ class Validator
         'datetimetz',
         'datetimetz_immutable',
         'timestamp',
+		'datetimeMS'
```

Comment:
This type does not seem to be supported by ["doctrine/dbal" 3.x](https://github.com/doctrine/dbal/blob/4534d445068f439d9ec287960ed7525f17b55840/src/Types/Types.php#L12-L36). Where is it declared?